### PR TITLE
Update Rasa NLU to Tensorflow 2.0

### PR DESCRIPTION
**Pull request by tabergma**
**Proposed changes**:
- Move to Tensorflow 2.0 for Rasa NLU

part of https://github.com/RasaHQ/research/issues/62#issuecomment-575626118

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -37,15 +37,10 @@
     "pymongo[tls,srv]~=3.8",
     "numpy~=1.16",
     "scipy~=1.2",
-    "tensorflow~=1.15.0",
+    "tensorflow~=2.0",
```
Comment (Ghostvv) [inline setup.py]:
```suggestion
    "tensorflow~=2.1",
```

File: rasa/nlu/featurizers/dense_featurizer/pretrained_lm_featurizer.py [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+    # "gpt2": False,
+    # "transfo-xl-wt103": False,
+    # "xlnet-base-cased": True,
+    # "xlm-mlm-enfr-1024": True,
+    # "distilbert-base-uncased": True,
+    # "roberta-base": True,
+}
+
+
+class PreTrainedLMFeaturizer(Featurizer):
```
Comment (tabergma) [inline rasa/nlu/featurizers/dense_featurizer/pretrained_lm_featurizer.py]: Should be removed for now.

File: rasa/nlu/test.py [hunk 3]
```diff
@@ -1114,6 +1114,10 @@ def run_evaluation(
             entity_results, extractors, output_directory, successes, errors
         )
 
+        write_prediction_for_hermit_evaluation(
```
Comment (tabergma) [inline rasa/nlu/test.py]: Delete

File: rasa/nlu/test.py [hunk 4]
```diff
@@ -1550,6 +1554,58 @@ def return_entity_results(results: EntityMetrics, dataset_name: Text) -> None:
         return_results(result, dataset_name)
 
 
+def write_prediction_for_hermit_evaluation(
```
Comment (tabergma) [inline rasa/nlu/test.py]: Delete

File: rasa/nlu/tokenizers/pretrained_lm_tokenizer.py [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+    # "gpt2": False,
+    # "transfo-xl-wt103": False,
+    # "xlnet-base-cased": True,
+    # "xlm-mlm-enfr-1024": True,
+    # "distilbert-base-uncased": True,
+    # "roberta-base": True,
+}
+
+
+class PreTrainedLMTokenizer(Tokenizer):
```
Comment (tabergma) [inline rasa/nlu/tokenizers/pretrained_lm_tokenizer.py]: Delete

**Pull request merged**